system_reports/MACOS/M1/osx-xcode-14.3.x-edge.log

```diff
@@ -40,42 +39,38 @@
 
 == All Nodejs versions selectable via asdf local nodejs x ==
 * Node.js default version: v20.4.0
-  18.16.1
-  18.16
+  18.17.1
+  18.17
   18
   20.4.0
  *20.4
   20
-18 => 18.16.1
-18.16 => 18.16.1
+18 => 18.17.1
+18.17 => 18.17.1
 20 => 20.4.0
 20.4 => 20.4.0
-lts => 18.16.0
+lts => 18.17.0
 * NPM: 9.7.2
 * Yarn: 1.22.19
 ========================================
 
 * Java:   system
   1.8
-  1.8.0.352
+  1.8.0.382
 * 11.0 (set by /Users/vagrant/.jenv/version)
-  11.0.17
+  11.0.20
   17.0
-  17.0.5
+  17.0.8
   8.0
-  temurin64-1.8.0.352
-  temurin64-11.0.17
-  temurin64-17.0.5
-* Kotlin: Kotlin version 1.9.0-release-358 (JRE 11.0.17+8)
+  temurin64-1.8.0.382
+  temurin64-11.0.20
+  temurin64-17.0.8
+* Kotlin: Kotlin version 1.9.0-release-358 (JRE 11.0.20+8)
 * Flutter: Flutter 3.7.12 • channel stable • https://github.com/flutter/flutter.git
 * fvm: 2.4.1
 * Flutter SDKs via fvm: Cache Directory:  /Users/vagrant/fvm/versions
 
 3.7.12 (global)
-3.7.6
```

Comment:
We never really deleted the old Flutter versions when we removed the definitions from the build tasks